{{cookiecutter.github_project_name}}/js/lib/labplugin.js

```diff
@@ -7,8 +7,8 @@ module.exports = {
   activate: function(app, widgets) {
       widgets.registerWidget({
           name: '{{ cookiecutter.npm_package_name }}',
-          version: {{ cookiecutter.npm_package_name }}.version,
-          exports: {{ cookiecutter.npm_package_name }}
+          version: plugin.version,
+          exports: plugin
```

Comment:
I also came across this today! First experience with `jupyter-widget-testing` as the `npm_package_name` results in an error.